pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -300,3 +307,17 @@ func deploymentCopy(rc *kapi.ReplicationController) (*kapi.ReplicationController
 	}
 	return copied, nil
 }
+
+func (c *DeploymentConfigController) handleErr(err error, key interface{}) {
+	if _, isFatal := err.(fatalError); isFatal {
```

Comment:
No, we don't surface those in the dc status. Opened https://github.com/openshift/origin/issues/9435